ocs_ci/utility/utils.py

```diff
@@ -2579,9 +2579,6 @@ def create_rhelpod(namespace, pod_name, timeout=300):
     """
     # importing here to avoid dependencies
     from ocs_ci.helpers import helpers
-    from ocs_ci.ocs.utils import label_pod_security_admission
-
-    label_pod_security_admission(namespace=namespace)
```

Comment:
I think that you will need to verify with AWS RHEL UPI deployment before merging. For RHEL deployment we are using default project NS and the pod needs to have privileged access which I believe is not handled by OLM in this case and will fail if we are missing this piece of code.  So please verify this before removing.

You can check: 
https://url.corp.redhat.com/f3a9460
Which was used to verify https://github.com/red-hat-storage/ocs-ci/pull/6564

Reply:
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/3719/
@petr-balogh